packages/react-core/src/components/Hint/examples/Hint.md

```diff
@@ -20,3 +20,15 @@ import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-ico
 ```ts file="HintBasicWithoutTitle.tsx"
 
 ```
+
+### Basic without no actions offset
```

Comment:
```suggestion
### With actions
```
Do you think this could just be called "with actions", since that's the ultimate concept that the example is supporting? Or is it important to specifically call out the prop here

Reply:
I think this makes sense 👍